ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml

```diff
@@ -119,6 +119,22 @@ tests:
   steps:
     cluster_profile: packet
     workflow: baremetalds-assisted-operator
+- as: e2e-metal-assisted-operator-bmh
+  steps:
+    cluster_profile: packet
+    env:
+      DEVSCRIPTS_CONFIG: |
+        IP_STACK=v4
```

Comment:
You may want to add:

```
export PROVISIONING_NETWORK_PROFILE=Disabled
```

Without this, the virtual media device won't start because of a missing implementation. There's work in progress to solve this. `KNIDEPLOY-4551`

A workaround would be to create static routes but I don't think that's worth it for this job